vote_text_drafts/rpip-30.md

```diff
@@ -0,0 +1,57 @@
+> Vote options:
+> - For
+> - Against
+> - Abstain
+
+# RPL Staking Rework (RPIP-30)
+
+This vote aims to align RPL reward spend with rETH capacity created.
+
+### Summary
+- RPL rewards will scale differently, with a focus on aligning RPL rewards with supplying rETH
+- It will be possible to withdraw down to 15% borrowed ETH
```

Comment:
What's the before after here? 10% to 15%? Might be worth including?

Reply:
Ah, the 10% deposit is unchanged?

Reply:
Down from 150% of bonded ETH. Will include.